national-resources/TEMPLATE_resources.md

```diff
@@ -0,0 +1,70 @@
+ref_to_main_resources: 
+  -  <!---REPLACE THIS with the tool name--->
+
+# List here tools and resources mainly relevant for the specific country
+national_resources: 
+  - name: <!---REPLACE THIS with the national resources about RDM in life sciences such as local instances of tools, guidelines or regulations--->
+    description:
+    how_to_access: <!--- REPLACE THIS free text to explain if credentials, login, specific affiliations etc., are needed to access the resource or tool--->
+    instance_of: <!--- REPLACE THIS with the tool name of which this resource is an instance of, taken from the all tools and resources page --->
+    related_pages:
+        # More information on which page_id you can use can be found at https://www.infectious-diseases-toolkit.org/contribute/website_overview 
+        human_biomolecular_data: [<!---REPLACE THIS with the page IDs of the human_biomolecular_data pages that you want to list here as related pages--->]
+        human_clinical_and_health_data: [<!---REPLACE THIS with the page IDs of the human_clinical_and_health_data pages that you want to list here as related pages--->]
+        socioeconomic_data: [<!---REPLACE THIS with the page IDs of the socioeconomic_data pages that you want to list here as related pages--->]
+        pathogen_characterisation: [<!---REPLACE THIS with the page IDs of the pathogen_characterisation pages that you want to list here as related pages--->]
+        showcase: [<!---REPLACE THIS with the page IDs of the showcase pages that you want to list here as related pages--->]
+    url:
+    registry:
+      biotools: <!--- DELETE ME if not needed --->
+      fairsharing: <!--- DELETE ME if not needed --->
+      tess: <!--- DELETE ME if not needed --->
+# More information on how to fill in this metadata section can be found here https://www.infectious-diseases-toolkit.org/contribute/page-metadata
+---
+
+<!-- Please take in mind our style guide https://www.infectious-diseases-toolkit.org/contribute/style_guide when writing the content of this page. -->
+
+<!---All the resources added above will appear on the table at the bottom of the page--->
+
+<!---Following information for the page text--->
+<!---If the information is already in another resource, please include the link instead of duplicating information--->
+<!---Please focus on resources that are relevant for the whole country for infectious diseases--->
+
+## Introduction 
+<!---General Infectious diseases data considerations for your country--->
+
+## Health authorities
+<!--- A section to list and provide context to agencies/authorities/institutions which define public health measures and policies --->
+
+## Disease surveillance initiatives
+
+## Dashboards and visualization platforms
+
+## National data sources
+<!--- A section to list and provide context to national data sources. Data source as defined in BY-COVID --->
```

Comment:
```suggestion
<!--- A section to list and provide context to national data sources.  In the context of BY-COVID, a data source can be a repository which should include at least the metadata and ideally the data, that might not be directly available when considering sensitive data. Also, repositories should have the capacity to share this data and therefore have a governance model in place on how to do it. It can also include registries of data sources important for the field, with a direct link to the original data sources to be able to request access to the data. --->
```

Based on the input from Laura in #134